hawk/src/test/java/com/orhanobut/hawk/HawkTest.java

```diff
@@ -44,7 +46,7 @@ public HawkTest() {
     context = Robolectric.buildActivity(Activity.class).create().get();
   }
 
-  @Override public void setUp() throws Exception {
+  @Override @Before public void setUp() throws Exception {
```

Comment:
No need to put before/after annotation since the class is extended from TestCase, We can simply remove the inheritance from here and use the annotation, I favor annotations over inheritance in test. Don't remember why I've extended it at the first place